raw_action.go

```diff
@@ -11,22 +11,25 @@ import (
 
 type RawAction struct {
 	BaseAction `yaml:",inline"`
+	Origin     string // there te source comes from
 	Offset     string
-	Source     string
-	Path       string
```

Comment:
We need to keep this in to not break current recipes

Reply:
We need to keep backwards compatibility with Source + Path usage. As suggested in a previous commit, you can use the Verify action to detect that usage (iotw Path is not "") and set Source = Path and Origin = filesystem afterwhich things would flow.

Would make sense to also print a note sayig that usage is deprecated

Reply:
done

Reply:
Please squash those patches together, we shouldn't break something in a patch and then fix it in the next as that will make bisecting nasty

Reply:
done